### ISSUE TITLE
symcc_afl build failed

### ISSUE THREAD

**Issue by johwi801** [HexHive/magma#84]
Hi, I'm having issues building symcc_afl with target libpng, It complains the cmake version is too low.

```
#17 887.8 /magma/fuzzers/symcc_afl/symcc
#17 887.8 CMake Error at CMakeLists.txt:3 (cmake_minimum_required):
#17 887.8   CMake 3.13.4 or higher is required.  You are running version 3.10.2
#17 887.8 
#17 887.8 
#17 887.8 -- Configuring incomplete, errors occurred!
#17 ERROR: executor failed running [/bin/sh -c ${FUZZER}/fetch.sh && ${FUZZER}/build.sh]: exit code: 1
```

full build log:
[symcc_afl_libpng_build.log](https://github.com/HexHive/magma/files/6986064/symcc_afl_libpng_build.log)

**Comment by adrianherrera** [comment]
Hi @johwi801, this has been fixed in the dev branch. I'll need to port the fix across to the v1.2 branch.

Thanks for the bug report!

**Comment by adrianherrera** [comment]
Note this is now also fixed in the latest release (v1.2) branch.

**Comment by johwi801** [comment]
Thank you for the quick fix!
